app/controllers/request-to-go-live/organisation-address/post.controller.js

```diff
@@ -2,99 +2,136 @@
-const ADDRESS_POSTCODE_FIELD = 'address-postcode'
-const TELEPHONE_NUMBER_FIELD = 'telephone-number'
+const clientFormIds = {
+  addressLine1: 'address-line1',
+  addressLine2: 'address-line2',
+  addressCity: 'address-city',
+  addressPostcode: 'address-postcode',
+  addressCountry: 'address-country',
+  telephoneNumber: 'telephone-number'
+}
 
 const validationRules = [
   {
-    field: ADDRESS_LINE1_FIELD,
+    field: clientFormIds.addressLine1,
     validator: validateMandatoryField,
     maxLength: 200
   },
   {
-    field: ADDRESS_LINE2_FIELD,
+    field: clientFormIds.addressLine2,
     validator: validateOptionalField,
     maxLength: 200
   },
   {
-    field: ADDRESS_CITY_FIELD,
+    field: clientFormIds.addressCity,
     validator: validateMandatoryField,
     maxLength: 100
   },
   {
-    field: ADDRESS_COUNTRY_FIELD,
+    field: clientFormIds.addressCountry,
     validator: validateMandatoryField,
     maxLength: 2
   },
   {
-    field: TELEPHONE_NUMBER_FIELD,
+    field: clientFormIds.telephoneNumber,
     validator: validatePhoneNumber
   }
 ]
 
 const trimField = (key, store) => lodash.get(store, key, '').trim()
 
-const validate = function validate (formFields) {
+const normaliseForm = (formBody) => {
+  const fields = [
+    clientFormIds.addressLine1,
+    clientFormIds.addressLine2,
+    clientFormIds.addressCity,
+    clientFormIds.addressCountry,
+    clientFormIds.addressPostcode,
+    clientFormIds.telephoneNumber
+  ]
+  return fields.reduce((form, field) => {
+    form[field] = trimField(field, formBody)
+    return form
+  }, {})
+}
+
+const validateForm = function validate (form) {
   const errors = validationRules.reduce((errors, validationRule) => {
-    const value = formFields[validationRule.field]
+    const value = form[validationRule.field]
     const validationResponse = validationRule.validator(value, validationRule.maxLength)
     if (!validationResponse.valid) {
       errors[validationRule.field] = validationResponse.message
     }
     return errors
   }, {})
 
-  const postCode = formFields[ADDRESS_POSTCODE_FIELD]
-  const country = formFields[ADDRESS_COUNTRY_FIELD]
+  const postCode = form[clientFormIds.addressPostcode]
+  const country = form[clientFormIds.addressCountry]
   const postCodeValidResponse = validatePostcode(postCode, country)
   if (!postCodeValidResponse.valid) {
-    errors[ADDRESS_POSTCODE_FIELD] = postCodeValidResponse.message
+    errors[clientFormIds.addressPostcode] = postCodeValidResponse.message
   }
   return errors
 }
 
-module.exports = (req, res) => {
-  const fields = [
-    ADDRESS_LINE1_FIELD,
-    ADDRESS_LINE2_FIELD,
-    ADDRESS_CITY_FIELD,
-    ADDRESS_COUNTRY_FIELD,
-    ADDRESS_POSTCODE_FIELD,
-    TELEPHONE_NUMBER_FIELD
-  ]
-  const formFields = fields.reduce((form, field) => {
-    form[field] = trimField(field, req.body)
-    return form
-  }, {})
+const submitForm = async function (form, serviceExternalId, correlationId) {
```

Comment:
This is a really clean design, nice! Splits apart logic and deals with specific parts of the request (bonus for handling Promises well)